### PR TITLE
prow-deploy: use quay image for kind test cluster

### PR DESCRIPTION
**Pull request by fgimenez**
This will prevent errors like https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/854/pull-project-infra-prow-deploy-test/1363409299254022145

/cc @dhiller @oshoval 

Signed-off-by: Federico Gimenez <fgimenez@redhat.com>

### CODE REVIEW

File: github/ci/prow-deploy/molecule/default/prepare.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@
     - name: Launch cluster
       shell: |
         kind delete cluster
-        kind create cluster --config {{ kind_cluster_config }}
+        kind create cluster --config {{ kind_cluster_config }} --image quay.io/kubevirtci/kind:v1.19.1
```
Comment (oshoval) [inline github/ci/prow-deploy/molecule/default/prepare.yml:57]: nice, do we need the 1.19 parameterized ?

Comment (fgimenez) [reply github/ci/prow-deploy/molecule/default/prepare.yml]: Makes sense, done.

**Review by oshoval — APPROVED** [review]
Thanks

**Review by dhiller — APPROVED** [review]
/lgtm
/approve

**Pull request merged**